docs/lifecycle.dot

```diff
@@ -148,7 +149,8 @@ digraph Metaflow {
     /* package */
     validate_dag -> init_environment
     init_environment -> package_init
-    package_init -> add_to_package
+    package_init -> add_custom_package
+    add_custom_package -> add_to_package
```

Comment:
@valayDave Can you also regenerate the new image?

Reply:
Done.